sections/04_study.md

```diff
@@ -82,7 +82,97 @@ use interesting network architectures to approach single-cell data.
+however, other methods based on interpolated Markov models [Salzberg] have
+performed better.  Due to the complexity of the problem, neural networks have
+been applied more to gene annotation (e.g. Orphelia [Hoff]), which usually
+have plenty of training examples.  Representations (similar to Word2Vec [ref]
+in natural language processing) for protein family classification has been
+introduced and classified with a skip-gram neural network [Asgari]. 
+Recurrent neural networks show good performance for homology and protein
+family identification [Hochreiter, Sonderby].  Interestingly, Hochreiter, who
+invented Long Short Term Memory, delved into homology/protein family
+classification in 2007, and therefore, deep learning is deeply rooted in
+functional classification methods.
+
+Most neural networks are being used for short sequence->taxa/function
+classification, where there is a lot of data for training (and thus suitable
+for NNs).  And, as a short side note, recurrent neural networks are showing
+success for base-calling (and thus removing the large error in the
+measurement of a pore's current signal) for the relatively new Oxford
+Nanopore sequencer [Boza].  However, due to small nubmers of metagenomic
+samples in studies, neural network uses for classifying phenotype from
+microbial composition are just beginning.   A standard MLP was able to
+classify wound severity from microbial species present in the wound [Kizek,
+2015].  Recently, multi-layer, recurrent networks (and convolutional
+networks) have been applied to microbiome genotype-phenotype, with Ditzler et
+al. being the first to associate soil samples with pH level using multi-layer
+perceptrons, deep-belief networks, and recursive neural networks (RNNs) 
+[Ditzler] .  Besides classifying the samples appropriately, Ditzler shows
+that internal phylogenetic tree nodes inferred by the networks are
+appropriate features representing low/high pH, which can provide additional
+useful information and new features for future metagenomic sample comparison.
+ Also, an initial study has show promise of these networks for diagnosing
+disease [Faruqi].  However, deep neural networks are not ideal for such
+problems since there are tens of samples (~20->40) available and
+hundreds/thousands of features (aka species).  Such underdetermined problems
+are still a challenge for deep neural networks that require many more
+training examples than features to sufficiently converge the weights on the
+hidden layers.
+
+In fact, due to convergence issues of neural networks, one would think
+that taxonomic classification would be impossible for deep neural networks. 
+However, with the 16S rRNA having hundreds of thousands of full-sequenced
+examples (compared to several thousand fully-sequenced whole-genome
+sequences), deep neural networks have been successfully applied to taxonomic
+classification of 16S rRNA genes, with convolutional networks outperforming
+RNNs and even random forests [Mrzelj].
```

Comment:
If you were going to speculate, is there any analysis that deep neural networks could enable that isn't yet in use?